rdt/transformers/base.py

```diff
@@ -212,10 +213,18 @@ def _build_output_columns(self, data):
         # make sure none of the generated `output_columns` exists in the data,
         # except when a column generates another with the same name
         output_columns = set(self.output_columns) - set(self.columns)
-        while set(output_columns) & set(data.columns):
+        repeated_columns = set(output_columns) & set(data.columns)
+        while repeated_columns:
```

Comment:
@amontanez24 It seemed more annoying, which is why I didn't bother. It should also never really happen (I mean, you have to have the columns `colum_name.is_null` AND `column_name#.is_null` in your data, which seems a little bit of a stretch), as you mentioned.

Also, it's not totally unreasonable to warn every time, since the "error" is occurring because of different columns, so it makes some sense to first tell you that the `colum_name.is_null` is suspicious, and if you also happen to have `column_name#.is_null` in your data, it should raise a separate warning telling you that this other, independent column is also suspicious.